inst/extdata/variable_details.csv

```diff
@@ -1553,4 +1553,24 @@ SMKG207_cont,N/A,cont,"cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p,
 SMKG207_cont,N/A,cont,"cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2005_p::SMKEG207, [SMKG207]",cat,NA::b,N/A,missing,missing,years,"[97,99]",don't know (97); refusal (98); not stated (99),agecigfd,Age started to smoke daily - former daily (G),
 SMKG207_cont,N/A,cont,"cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2005_p::SMKEG207, [SMKG207]",cat,NA::b,N/A,missing,missing,years,else,else,agecigfd,Age started to smoke daily - former daily (G),
 WTS_M,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::WTSAM, cchs2003_p::WTSC_M, cchs2005_p::WTSE_M, [WTS_M]",N/A,copy,N/A,N/A,N/A,N/A,"[1.17,26332.62]",N/A,Weights,Weights - Master,
-id_year,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[data_name, ADM_RNO]",N/A,id_from::unite,N/A,N/A,N/A,N/A,N/A,N/A,Unique ID,Unique ID,Variable appends data_name to sequential random number to generate a unique identifier for all respondents 
\ No newline at end of file
+id_year,N/A,cont,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","DerivedVar::[data_name, ADM_RNO]",N/A,id_from::unite,N/A,N/A,N/A,N/A,N/A,N/A,Unique ID,Unique ID,Variable appends data_name to sequential random number to generate a unique identifier for all respondents 
+LBFA_31A,LBFA_31A_cat9_1,cat,cchs2001_p,[LBFA_31A],cat,1,9,Management,Management,N/A,1,Management,Occupation,Main job or business,

```

Comment:
I tend to keep CCHS variables in alphabetical order just for ease of access to the user.

Reply:
Yeah, sorry. I was going to do that after it got approved so that the PR would not be hard to review.